### DISCUSSION TITLE
There was an error committing your changes: File could not be edited

### DISCUSSION THREAD

**Discussion by comisac01** [community/community#112986] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Getting this error for repository if i am changing html files of my project, please suggest resolution
![Capture](https://github.com/community/community/assets/120954337/88eb45cd-976a-4820-b781-9def66278eb0)

**Comment by yuzurupupu** [comment]
I also encountered this problem, it seems that this error occurs because the file is too large to exceed this range.

**Comment by ShaiddinShaik** [comment]
What's the solution for this issue.
